src/Microsoft.Identity.Client/ApiConfig/AcquireTokenInteractiveParameterBuilder.cs

```diff
@@ -101,6 +101,22 @@ public AcquireTokenInteractiveParameterBuilder WithUseEmbeddedWebView(bool useEm
             return this;
         }
 
+// Default browser WebUI is not available on mobile (Android, iOS, UWP), but allow it at runtime
+// to avoid MissingMethodException
+#if NET_CORE || NETSTANDARD || DESKTOP || RUNTIME
+        /// <summary>
+        /// Specifies options for using the system OS browser handle interactive authentication.
+        /// </summary>
+        /// <param name="options">Data object with options</param>
+        /// <returns>The builder to chain the .With methods</returns>
+        public AcquireTokenInteractiveParameterBuilder WithSystemWebViewOptions(SystemWebViewOptions options)
+        {
+            CommonParameters.AddApiTelemetryFeature(ApiTelemetryFeature.WithDefaultBrowserOptions);
```

Comment:
since they're sending in their own options, are these really the "default browser options"?  i know our telemetry story isn't solidified yet but this seems somewhat at odds with what is actually happening here.

Reply:
I am not sure what you are suggesting here. Is it the naming?

Reply:
Ah ok, I think I understand now. These are not default options, but options for the default browser. I'll change the name.